Tasks/JavaToolInstallerV0/build_task.sh

```diff
@@ -0,0 +1 @@
+node make.js build --task JavaToolInstallerV0 && tfx build tasks delete --task-id C0E0B74F-0931-47C7-AC27-7C5A19456A36 && tfx build tasks upload --task-path .\_build\Tasks\JavaToolInstallerV0
```

Comment:
Is this file added intentionally?

Reply:
I think it's better to remove it, thanks